{"name": "Signup using invalid mobile number1", "status": "broken", "statusDetails": {"message": "AttributeError: 'Context' object has no attribute 'assertion_comparison_otp_page'\n", "trace": "  File \"C:\\Users\\RAMPRK\\PycharmProjects\\pythonBDD\\venv\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\RAMPRK\\PycharmProjects\\pythonBDD\\venv\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\signin.py\", line 52, in step_impl\n    if context.assertion_comparison_otp_page.__eq__(\"Verify your mobile number\"):\n       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\RAMPRK\\PycharmProjects\\pythonBDD\\venv\\Lib\\site-packages\\behave\\runner.py\", line 321, in __getattr__\n    raise AttributeError(msg)\n"}, "steps": [{"name": "Given The browser is opened and the website is loaded", "status": "passed", "start": 1678777472666, "stop": 1678777483764}, {"name": "When The Login button on the website is clicked", "status": "passed", "start": 1678777483764, "stop": 1678777486436}, {"name": "When Create your account is selected in the Login menu", "status": "passed", "start": 1678777486436, "stop": 1678777488576}, {"name": "When The country dropdown gets clicked", "status": "passed", "start": 1678777488577, "stop": 1678777490761}, {"name": "When India with the appropriate mobile extension is selected", "status": "passed", "start": 1678777490762, "stop": 1678777492875}, {"name": "Then The invalid mobile number sequence is entered", "status": "passed", "start": 1678777492875, "stop": 1678777493122}, {"name": "And The Continue button is clicked", "status": "passed", "start": 1678777493122, "stop": 1678777495225}, {"name": "Then It should stay in the same page with a warning prompt", "status": "broken", "statusDetails": {"message": "AttributeError: 'Context' object has no attribute 'assertion_comparison_otp_page'\n", "trace": "  File \"C:\\Users\\RAMPRK\\PycharmProjects\\pythonBDD\\venv\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\RAMPRK\\PycharmProjects\\pythonBDD\\venv\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\signin.py\", line 52, in step_impl\n    if context.assertion_comparison_otp_page.__eq__(\"Verify your mobile number\"):\n       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\RAMPRK\\PycharmProjects\\pythonBDD\\venv\\Lib\\site-packages\\behave\\runner.py\", line 321, in __getattr__\n    raise AttributeError(msg)\n"}, "start": 1678777495225, "stop": 1678777495230}], "start": 1678777472663, "stop": 1678777497596, "uuid": "20d53044-5364-4255-af2a-0a36e406cf92", "historyId": "57d9d975fb92e6825e5960faee4bf65a", "fullName": "Signin module: Signup using invalid mobile number1", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Signin module"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}